### ISSUE TITLE
Facebook OAuthException #100 - Owners of URL

### ISSUE THREAD

**Issue by IndigoViking** [engram-design/SocialPoster#23]
### Description
Trying to  post entry to Facebook and getting this response:

```
{
    "error": {
        "message": "(#100) Only owners of the URL have the ability to specify the picture, name, thumbnail or description params.",
        "type": "OAuthException",
        "code": 100,
        "fbtrace_id": "E5hE8RS\/Ie6"
    }
}
```

### Steps to reproduce

1. Install Social Poster, setup app, connect to Facebook
2. Try to post

### Additional info

- Plugin version: 2.0.4
- Craft version: Craft Pro 3.1.18

**Comment by engram-design** [comment]
So setting the `picture` on a post to Facebook is deprecated, and this effects posts where you've set an image to be posted to Facebook.

> "As of November 7, 2017, link customization is available however the link must be owned by the posting page and a page access token is required. To verify ownership, check the ownership_permissions{can_customize_link_posts} field on the URL node. See our Link Ownership Guide for more information. For versions 2.10 and lower, picture, name, thumbnail, and description are deprecated. caption is deprecated for all versions."

In order to manually post an image alongside your post, you'll need to complete [Domain Verification](https://developers.facebook.com/docs/sharing/domain-verification).

Otherwise, I'd recommend leaving out the `Image for Post` field, and ensure that your entry is setup with the correct `og:image` meta tags to use that image. When posting to Facebook, it'll look for this image for the URL you're specifying and use that as the picture.

The only alternative is for us to change so that it uploads the asset to your Facebook Page as a photo - but then I wouldn't want to clutter your Page's photos...

Another frustrating restriction on Facebook's end.

**Comment by IndigoViking** [comment]
I've removed the field associated with the Image for Post and I removed it from the sidebar but I'm still getting the same message.

**Comment by engram-design** [comment]
Should be fixed in [2.0.5](https://github.com/verbb/social-poster/releases/tag/2.0.5)

**Comment by IndigoViking** [comment]
Got it working thanks!
